code/fmriprep/ss-fmriprep-anatonly.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#SBATCH --partition=russpold
+#SBATCH --mem=30GB
+#SBATCH --cpus-per-task=4
+#SBATCH --time=2-00:00:00
+#SBATCH --job-name=fmriprep
+#SBATCH --error="slurm-%A_%a.err"
+
+# Run only the anatomical workflow of fMRIPrep
+
+DATADIR="/oak/stanford/groups/russpold/inprocess/cprovins/hcph-pilot/"
+SUB="sub-pilot"
+STUDY=`basename $DATADIR`
+
+echo "Processing the anatomical workflow on subject: $SUB"
+
+IMG="/oak/stanford/groups/russpold/users/cprovins/singularity_images/fmriprep-23.1.4.simg"
+
+WORKDIR="${L_SCRATCH}/fmriprep/${STUDY}/"
+mkdir -p ${WORKDIR}
+OUTDIR="${DATADIR}/derivatives"
+mkdir -p $OUTDIR
+
+PATCHES=""
+
+BINDINGS="-B $DATADIR/inputs:/data:ro \
+-B ${WORKDIR}:/work \
+-B ${OUTDIR}:/out \
+-B `pwd`/license.txt:/opt/freesurfer/license.txt \
+$PATCHES"
+
+FMRIPREP_CMD="/data /out/fmriprep-23.1.4 participant \
+-w /work --anat-only --skip_bids_validation \
+--nprocs 4 --mem 25G --omp-nthreads 2 \
+-vv"
```

Comment:
On this line, I would suggest the `--omp-nthreads` option is set to 8 or higher (12, 16?). The reason being the anatomical workflow will run much faster and you are unlikely to fall into memory issues.

Reply:
I'm concerned where the file will be written out - It doesn't seem like it will be picked up by the command line (which looks incomplete in terms of bound folders).

Reply:
True - I blindly copied the implementation from `code/fmriprep/ss-fmriprep.sh`. I suggested changes to the binding accordingly. As for where the fill will be written, I haven't tested it myself, maybe @celprov has more insights on that.

Reply:
Originally, the code is stored in a `code` subfolder in the dataset, i.e under `DATASETPATH/hcph-pilot/code`.
I copied the code over to the `hcph-sops` repo to incorporate it in the SOP.

The filter files would then be written under `DATASETPATH/hcph-pilot/code`, the same folder that contains the code.